.github/workflows/test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: Install dependencies
-      run: dart pub get
+      run: dart dart pub get
```

Comment:
No need to duplicate the `dart` command. That won't work in GitHub Actions.

I think you just need a copy/paste, so just need to correct this.

Reply:
My apologies, I caught that mistake and did a second commit that reflects that change.